index.html

```diff
@@ -2,213 +2,260 @@
-                <li class="tech">Codepen</li>
-                <li class="tech">Gitlab</li>
-                <li class="tech">Terminal</li>
-              </ul>
-            </div>
-        </section>
-      
-    </main>
-  </body>
+        </div>
+        <hr class="about-separater" />
+        <div>
+          <ul class="technologies">
+            <li class="tech-title font-inter">
+              <img src="./Skeleton/diamond.png" alt="diamond" />
+              <div class="text-white">Language</div>
+            </li>
+            <li class="tech">Javascript</li>
+            <li class="tech">Ruby</li>
+            <li class="tech">HTML</li>
+            <li class="tech">Css</li>
+          </ul>
+          <ul class="technologies">
+            <li class="tech-title">
+              <img src="Skeleton/Rectangle.png" alt="rectangle" />
+              <div class="text-white">Frameworks</div>
+            </li>
+            <li class="tech">Bootstrap</li>
+            <li class="tech">Roby on Rails</li>
+            <li class="tech">RSPec</li>
+            <li class="tech">CapyBara</li>
+            <li class="tech">Selenium</li>
+          </ul>
+          <ul class="technologies">
+            <li class="tech-title">
+              <img src="Skeleton/circle.png" alt="circle" />
+              <div class="text-white">Skills</div>
+            </li>
+            <li class="tech">Codekitt</li>
+            <li class="tech">Github</li>
+            <li class="tech">Codepen</li>
+            <li class="tech">Gitlab</li>
+            <li class="tech">Terminal</li>
+          </ul>
+        </div>
+      </section>
+      <section class="contact">
+        <form action="https://formspree.io/f/xlezlkyw" method="post">
+          <p class="form-detail">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+             <input type="text" placeholder="Full Name" maxlength="30" required>
+             <input type="email" placeholder="Email address" required>
+             <textarea name="msgt1" id="t1" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+          <p>
+            <button class="button" type="submit">Get in touch</button>
+           </p>
+      <footer class="social-footer-social">
+      <a href="#"  class="social-link">
+        <img class="social-icon" src="./icons/github.png" alt="GitHub"/>
+      </a>
+      <a href="#"  class="social-link">
+        <img class="social-icon" src="./icons/Linkedin.png" alt="LinkedIn"/>
+      </a>
+      <a href="#"  class="social-link">
+        <img class="social-icon" src="./icons/angel.png" alt="Angel"/>
+      </a>
+      <a href="#"  class="social-link">
+        <img class="social-icon" src="./icons/twitter.png" alt="Twitter"/>
+      </a>
+      <a href="#"  class="social-link">
+        <img class="social-icon" src="./icons/medium.png" alt="Medium"/>
+      </a>
+    </footer>
+  </form>
```

Comment:
- [ ] Let's try to be more organized in our code and obey the semantic principles. The `footer` tag should wrap the `form`, not the other way round. Kindly refactor accordingly. Ensure the same design is followed.

- [ ] The social media icons are not part of the form hence they should not be wrapped by the `form` tag. Kindly move the icons to be below the `form` element.